### ISSUE TITLE
Proper logging for backend

### ISSUE THREAD

**Issue by Soreine** [14Plumes/Hexode#12]
For now I'll just add colored output in case of errors. But we should think of using a cleaner logging module.

**Comment by KtorZ** [comment]
Indeed. Possibly something with persistence even if plenty of tools now are just used to parse the stdout output messages directly.
